src/app/pages/organization/companies/list/companies-list-table-data-source.ts

```diff
@@ -171,7 +177,7 @@ export class CompaniesListTableDataSource extends TableDataSource<Company> {
     const openInMaps = new TableOpenInMapsAction().getActionDef();
     openInMaps.disabled = !Utils.containsAddressGPSCoordinates(company.address);
     if (company.issuer) {
-      if (this.isAdmin) {
+      if (this.canCreateCompany && this.canReadCompany && this.canUpdateCompany && this.canDeleteCompany) {
         return [
           this.editAction,
```

Comment:
For the Edit button use canUpdate ELSE for the Display button check canRead.
You see my point, the type button should be displayed according the corresponding Auth.